app/routes/endemics/check-answers.js

```diff
@@ -29,27 +29,24 @@ const getBackLink = (isReview, isSheep) => {
     : `${urlPrefix}/${routes.endemicsBiosecurity}`
 }
 const getNoChangeRows = (
-  isReview,
-  isPigs,
-  isSheep,
-  typeOfLivestock,
+  { isReview, isPigs, isSheep, typeOfLivestock, dateOfVisit }, // claim info
```

Comment:
Could the other 3 arguments be named in this object too? Just helps prevent mistakes I think, when there are so many arguments